### PR TITLE
TYP: use OpsMixin for DecimalArray

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/extension/decimal/array.py [hunk 1]
```diff
@@ -197,6 +198,25 @@ def _reduce(self, name: str, skipna: bool = True, **kwargs):
             ) from err
         return op(axis=0)
 
+    def _cmp_method(self, other, op):
```
Comment (jreback) [inline pandas/tests/extension/decimal/array.py:201]: this looks generic, can it be *in* ExtensionScalarOpsMixin? (and maybe have that inherit OpsMixin)

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/extension/decimal/array.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/extension/decimal/array.py:201]: i tried that and it felt clunky, decided to make it a separate step

**Comment by jreback** [comment]
i guess should disable this for now
```
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/timezones.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/offsets.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/timestamps.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/ccalendar.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/conversion.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/np_datetime.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/timedeltas.pxd Imports are incorrectly sorted and/or formatted.
ERROR: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/period.pxd Imports are incorrectly sorted and/or formatted.
Check import format using isort DONE
```

**Comment by jbrockmendel** [comment]
> i guess should disable this for now

I thought i saw a PR that addressed that.  if i dont find it ill open one shortly

**Comment by jbrockmendel** [comment]
yah looks like someone already got to the isort thing, so this is green

**Review by jreback — APPROVED** [review]

**Pull request merged**